README.md

```diff
@@ -19,50 +19,70 @@ The OSHDB offers a distributed data base that splits storage and computation loa
+
 OpenStreetMap History Data
 --------------------------
 
-OpenStreetMap contains a large variety of geographic data, for example differing in scale (ranging from single points of interests to whole country borders), or feature type (from concrete things like buildings up to more abstract concepts such as turn restrictions) and offers a lot of metadata (about individual contributors, modifications, etc.) that can be analyzed in a multitude of ways. At the same time, possibilities of working with this data are limited because of the large amount of needed resources and the lack of an easy-to-use analysis software. The huge information treasure in OSM for researchers, journalists, community members and other interested people is thereby kept hidden. A central goal of the OSHDB is to make this treasure available for a larger public.
+OpenStreetMap contains a large variety of geographic data, differing widely in scale and feature type. OSM contains everything from single points of interests to whole country borders, from concrete things like buildings up to more abstract concepts such as turn restrictions. OSM also offers metadata about the history and the modifications that are made to the data, which can be analyzed in a multitude of ways.
+
+Because of it's size and variety, possibilities of working with OSM history data are limited and there exists a lack of an easy-to-use analysis software. A goal of the OSHDB is to make OSM data more accessible to researchers, data journalists, community members and other interested people.
 
 Central Concepts
 ----------------
 
-### Data Partitioning
+The OSHDB is designed to be appropriate for a large spectrum of potential use cases and is therefore built around the following central ideas and design goals:
+
+* _Lossless Information_: The full OSM history data set should be stored and be queryable by the OSHDB, including errorneous or partially incomplete data.
+* _Simple, Generic API_: Writing queries with the OSHDB should be simple, and intuitive, while at the same time flexbile and generic to allow a wide variety of analysis queries. 
+* _High Performance_: The OSM history data set is large and thus requires efficiency in the way the data is stored and in the way it can be accessed and processed. 
+* _Local and Distributed Deployment_: Analysis queries should scale well from data explorations of small regions up to global studies of the complete OSM data set.
+
+### Data Model
+
+The OSHDB uses it's own binary data format that encapsulates the OSM history data available from [planet.osm.org](https://planet.osm.org/) and is optimized for efficiently storage and access to the history data. In order to allow parallelized data processing, the OSHDB data model also includes a data partitioning scheme.
```

Comment:
"... and is optimized for an efficient storage and access to the history data"
or
".. and is optimized for efficiently storing and accessing the history data"